flybywire-aircraft-a320-neo/layout.json

```diff
@@ -0,0 +1,3 @@
+{
+  "content": []
+}
```

Comment:
Is this file needed?

Reply:
It was generated alongside the other two .json files. I would hazard a guess to say that we do not need this file, considering the builder in-sim would have populated this if we would require.

Could you or someone else with the CRJ check to see if this file is included? (Or any other marketplace aircraft)

Reply:
CRJ has it - so I guess we should keep it